tests/test_tiebreaks.py

```diff
@@ -588,7 +590,7 @@ def test_buchholz_cut1(self):
     def test_adjusted_score(self):
         results = {
             player.id: tie_break.TieBreakUtils.adjusted_score(
-                player, self.tournament, 5
+                player, after_round=None,
```

Comment:
```suggestion
                player, after_round=self.tournament.rounds,
```
after_round has to be initialized for adjusted_score (tests are failing)